web/src/app/app.routes.ts

```diff
@@ -43,6 +44,11 @@ const routes: Routes = [
         pathMatch: 'full',
         component: HelpComponent,
       },
+      {
+        path: 'terms-and-conditions',
+        pathMatch: 'full',
```

Comment:
Not really an issue perhaps its good to have but the path will work fine without setting this, its more useful on things that are ambiguous and you want to do a redirect in certain cases

Reply:
Thanks for the feedback @andrew-cunliffe 👍  . That is something @chowbertilla and I did talk about. I kept it in there for consistency